conda/recipes/cuml/recipe.yaml

```diff
@@ -18,95 +18,105 @@ package:
 source:
   path: ../../..
 
-build:
-  string: cuda${{ cuda_major }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
-  files:
-    exclude:
-      - '*libarrow.so.*gdb.py'
-  prefix_detection:
-    # See https://github.com/rapidsai/build-planning/issues/160
-    # Blanket ignore here as there are quite a few shared objects shipped in cuml
-    ignore_binary_files: True
-  script:
-    content: |
-      ./build.sh cuml -v
-    secrets:
-      - AWS_ACCESS_KEY_ID
-      - AWS_SECRET_ACCESS_KEY
-      - AWS_SESSION_TOKEN
-    env:
-      CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER") }}
-      CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER") }}
-      CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER") }}
-      CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR") }}
-      SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET") }}
-      SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT") }}
-      SCCACHE_REGION: ${{ env.get("SCCACHE_REGION") }}
-      SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL") }}
-      SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS") }}
-      SCCACHE_S3_KEY_PREFIX: cuml/${{ env.get("RAPIDS_CONDA_ARCH") }}/cuda${{ cuda_major }}
+outputs:
```

Comment:
I have no idea if I did this correctly - I haven't written a conda metapackage recipe in a few years, and didn't try to run the build locally to see (I'm not actually sure how I would do that). This recipe could probably use a decent review.